packages/react/src/_availableComponents.js

```diff
@@ -0,0 +1,46 @@
+export default Object.freeze([
+  "ActionBar",
```

Comment:
Just curious? What is the benefit of doing this? I'm not sure I fully understand.

Reply:
See comment above :up: